src/test/java/com/org/jenkins/custom/jenkins/distribution/service/PluginControllerTest.java

```diff
@@ -0,0 +1,28 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import static org.junit.Assert.assertEquals;
+
+@SpringBootTest
+public class PluginControllerTest extends SpringMVCSetup {
+
+    @Override
+    @Before
+    public void setup() {
+        super.setup();
+    }
+
+    @Test
+    public void getPluginList() throws Exception {
+        String uri = "/api/plugin/getPluginList";
+        MvcResult mvcResult = mvc.perform(MockMvcRequestBuilders.get(uri).accept(MediaType.APPLICATION_JSON_VALUE)).andReturn();
+        int status = mvcResult.getResponse().getStatus();
+        assertEquals(200, status);
```

Comment:
I do not know the Spring Framework, but is it possible to actually test a non-empty list of plugins and check that the list obtained is as expected?

Reply:
I am not too sure we can try I guess

Reply:
This comment actually belongs to the review.